components/scream/src/share/scream_pack.hpp

```diff
@@ -46,13 +54,13 @@ using OnlyMask = typename std::enable_if<Mask::masktag,Mask>::type;
 template <typename Mask, typename Return>
 using OnlyMaskReturn = typename std::enable_if<Mask::masktag,Return>::type;
 
-#define scream_masked_loop(mask) \
+#define scream_masked_loop(mask, s)                         \
```

Comment:
To make the loop construct transparent, the index variable is now required. Was silly of me not to do this originally.